### DISCUSSION TITLE
Welcome to aitrack Discussions!

### DISCUSSION THREAD

**Discussion by AIRLegend** [AIRLegend/aitrack#81] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Semaj660** [comment]
I could be wrong here but does AI Track only use 30fps? My camera looks a lot smoother in the Droidcam preview in comparison to the AI Track preview but I initially assumed this was down to processing or something similar

**Comment by phobiadan** [comment]
It would be great to have a calibration feature of the head position in different spots to make tracking more precisely.

**Comment by christiansouteyrand** [comment]
Hi,
i don't found airtrack.exe in the folder  ?

**Comment by TerrenceMeiring** [comment]
Hey All

Has anyone done any research into adding an RGB camera like the Luxonis OAK-1 to AITrack?
